spec/rakuten_web_service/gora/course_detail_spec.rb

```diff
@@ -0,0 +1,61 @@
+# encoding: utf-8
+
+require 'spec_helper'
+require 'rakuten_web_service'
```

Comment:
please remove this `require` from all spec files. `require`ing `rakuten_web_service` in spec files causes SimpleCov fail to measure coverages. I got known it in this week...